philips-hue-adapter.js

```diff
@@ -210,6 +213,7 @@ class PhilipsHueDevice extends Device {
             '@type': 'OnOffProperty',
             label: 'On/Off',
             type: 'boolean',
+            readOnly: true,
```

Comment:
This should not be `readOnly`. That would cause the lights to no longer function. 😄

Reply:
That indeed was not an intended change, thanks for catching that.

Reply:
I'm assuming that change was supposed to be for the daylight sensor.